support/doc/tools.md

```diff
@@ -42,20 +41,12 @@ Clone the PeerTube repo to get the latest version (even if you are on your PeerT
 
 ```
 $ git clone https://github.com/Chocobozzz/PeerTube.git
-$ CLONE="$(pwd)/PeerTube"
+$ CLONE="$(pwd)/PeerTube && cd $CLONE"
 ```
 
-Run ``yarn install --pure-lockfile``
-```
-$ cd ${CLONE}
-$ yarn install --pure-lockfile
-```
+Run `yarn install --production --pure-lockfile` or `NOCLIENT=1 yarn install --production --pure-lockfile` if you wish to save space by not having dependencies for the web interface of PeerTube.
```

Comment:
If you build the server you need the dev dependencies, therefore you cannot use `--production`

And do you have a documentation link regarding the `NOCLIENT` env variable?

Reply:
Ah right, I put the `--production` from the point of view of an install using a release. Let me correct that.

As for `NOCLIENT`, I introduced it in https://github.com/Chocobozzz/PeerTube/commit/590924284c571d30a6b1852296b2043f3bfa5400.